controllers/operands/kubevirtConsolePlugin.go

```diff
@@ -281,6 +281,17 @@ func NewKvUIPluginSvc(hc *hcov1beta1.HyperConverged) *corev1.Service {
 	}
 }
 
+// TODO move to a new fileu
+func NewPasstBindingCNISA(hc *hcov1beta1.HyperConverged) *corev1.ServiceAccount {
```

Comment:
this (and the rest of the objects we deploy for passt)
will be moved to a new file
`controllers/operands/passt.go` which has the required function to generate the objects